analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnchangedLocalVariablesShouldBeConst.ToFix.cs

```diff
@@ -0,0 +1,35 @@
+﻿enum Colors { Red, Blue, Green };
+
+class ToReplace
+{
+    void Alias()
+    {
+        string str = "str";              // Noncompliant
+        char ch = 'c';                   // Noncompliant
+        byte bits = 8;                   // Noncompliant
+        sbyte sbits = 8;                 // Noncompliant
+        short int16 = 16;                // Noncompliant
+        ushort uint16 = 16;              // Noncompliant
+        int int32 = 32;                  // Noncompliant
+        uint uint32 = 32;                // Noncompliant
+        long int64 = 64;                 // Noncompliant
+        long longVal = 1;                // Noncompliant
+        ulong ulongVal = 1;              // Noncompliant
+        float single = 32;               // Noncompliant
+        double floating = 64;            // Noncompliant
+        decimal dec = 128;               // Noncompliant
```

Comment:
```suggestion
        decimal dec = 128;               // Noncompliant
        int i1 = 1,                      // Noncompliant
            i2 = 2;                      // Noncompliant
```

Reply:
If you provide a fix for this, you also need to add some trivia tests like e.g. so:

```cs
        /* Before type */ int /* After type */ i1 /* After i1 */ = 1 /* After initializer */,                      // Noncompliant
            i2 = 2;                      // Noncompliant
```

You also need a test for a declaration without initializer:

```cs
        int i1,
            i2 = 2;                      // Noncompliant
```